pkg/cosign/image_signer.go

```diff
@@ -210,40 +213,24 @@ func (s *ImageSigner) SignBuilder(
 			},
 		)
 
-		if _, found := os.LookupEnv(cosignutil.CosignDockerMediaTypesEnv); found {
-			err = os.Unsetenv(cosignutil.CosignDockerMediaTypesEnv)
+		if _, found := os.LookupEnv(CosignDockerMediaTypesEnv); found {
+			err = os.Unsetenv(CosignDockerMediaTypesEnv)
```

Comment:
Is this to prevent an end-user from injecting an incorrect media type through an environmental variable?